parser/SqlLexer.lpp

```diff
@@ -120,6 +120,7 @@ unsigned_numeric_literal {exact_numeric_literal}|{approximate_numeric_literal}
 "between"          return TOKEN_BETWEEN;
 "bigint"           return TOKEN_BIGINT;
 "bit"              return TOKEN_BIT;
+"bloom_filter"     return TOKEN_BLOOM_FILTER_INDEX;
```

Comment:
Thanks Craig for the reviews. I will update the token for bloom_filter and add a keyword for CSBTree as well. Yes, there is a working bloom filter implementation in one of the branches at University repo, I plan to integrate that with the Pivotal branch in coming weeks.